lib/puppet_litmus/serverspec_helpers.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+# helper functions for running puppet commands, and helpers
+module PuppetLitmus::ServerspecHelpers
```

Comment:
Going to do some bikeshedding: `PuppetLitmus::Helpers::Serverspec` perhaps?

Reply:
moved it to PuppetLitmus::Serverspec